src/app/reservation/[roomId]/page.tsx

```diff
@@ -0,0 +1,18 @@
+import Head from 'next/head'
+
+type RoomReservationParams = {
```

Comment:
usually we call it "props" not "params"

Reply:
in this instant the params are called correctly, but I think it would be better to create props in a single place instead of splitting props in two. So either:
```typescript
type RoomReservationProps = {
  params: {roomId: number}
}
```
or:
```typescript
export default function RoomReservation({params}:  {
  params: {roomId: number}
}) {}
```